core/domain/collection_domain.py

```diff
@@ -342,13 +394,25 @@ def to_yaml(self):
 
     @classmethod
     def _convert_v1_dict_to_v2_dict(cls, collection_dict):
-        """Converts a v1 collection dict into a v2 collection dict."""
+        """Converts a v1 collection dict into a v2 collection dict.
+
+        Adds a language code, and tags.
+        """
         collection_dict['schema_version'] = 2
         collection_dict['language_code'] = feconf.DEFAULT_LANGUAGE_CODE
         collection_dict['tags'] = []
         return collection_dict
 
     @classmethod
+    def _convert_v2_dict_to_v3_dict(cls, collection_dict):
```

Comment:
Perhaps we should refer directly to ``_convert_collection_contents_v2_dict_to_v3_dict`` in the doc string so readers are aware that this part of that migration.